Source/Core/VideoCommon/CommandProcessor.cpp

```diff
@@ -87,6 +87,8 @@ void CommandProcessorManager::DoState(PointerWrap& p)
   p.Do(m_cp_status_reg);
   p.Do(m_cp_ctrl_reg);
   p.Do(m_cp_clear_reg);
+  p.Do(m_perf_select);
+  p.Do(m_unk_0a_reg);
```

Comment:
I think you'll need to bump the state version?

Reply:
Ah, I keep forgetting that...